lib/Manager.php

```diff
@@ -121,7 +121,7 @@ public function createParticipantObject(Room $room, array $row) {
 			$lastMention = new \DateTime($row['last_mention']);
 		}
 
-		return new Participant($this->db, $room, $row['user_id'], (int) $row['participant_type'], (int) $row['last_ping'], $row['session_id'], (bool) $row['in_call'], (bool) $row['favorite'], $lastMention);
+		return new Participant($this->db, $room, (string) $row['user_id'], (int) $row['participant_type'], (int) $row['last_ping'], $row['session_id'], (bool) $row['in_call'], (bool) $row['favorite'], $lastMention);
```

Comment:
Is `$row['session_id']` always guaranteed to be a string or is the cast missing there?

Reply:
missed it, let's be sure